roxie/ccd/ccdserver.cpp

```diff
@@ -4446,10 +4446,16 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
         {
             if (!contextCached)
             {
+                //The following serialized the extra meta/tracing information that is associated with a request from the
+                //server to the worker.  Previously it ensured 0 ws never sent, but no logic relies on that anymore.
                 logInfo.clear();
-                unsigned char loggingFlags = LOGGING_FLAGSPRESENT | LOGGING_TRACELEVELSET;
                 unsigned char ctxTraceLevel = activity.queryLogCtx().queryTraceLevel() + 1; // Avoid passing a 0
                 const QueryOptions &options = ctx->queryOptions();
+                ISpan * activeSpan = queryThreadedActiveSpan();
+                // Trace and span id is 24 bytes.  Should we always send it or not, or only if recordingEvents?
```

Comment:
Only if we expect the workers to either propagate downstream (soapcalls, etc.) or perform an action with the info regardless of isSampled such as logging, alerts, etc.
Obviously the added overhead and its affect on performance should be considered as well.